test.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+import sys
+import os
+
+print(os.listdir("cache"))
```

Comment:
Probably didn't mean to check-in this file?

Reply:
No. Mistake on my part.